CDCgov/prime-reportstream
# 10486: support multiple file schemes when loading schemas

This PR adds functionality to resolve schemas via different schemes, including the file system, the classpath and azure.

Test Steps:
Run the smoketests

There is also test coverage and this should be a NOOP since no schemas are getting changed.  It should be possible to update all of ORU_R01 to use URIs, but that is actually just the work of another ticket.


## Changes
- Adds support for reading inputs when resolving schemas from azure, filesystem and the classpath

## Checklist

### Testing

### Process

## Linked Issues
- #10486 

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
